### PR TITLE
adding changed css for pinned blog

### PR DESCRIPTION
**Pull request by harsh-kr-jha**
adding CSS and HTML code to switch between class when pinned status change

### CODE REVIEW

File: src/app/page/blog/blog.component.css [hunk 1]
```diff
@@ -1,13 +1,43 @@
+.special-feed-card {
+    
+
```
Comment (pulsejet) [inline src/app/page/blog/blog.component.css]: Why is there a newline here?

File: src/app/page/blog/blog.component.css [hunk 2]
```diff
@@ -1,13 +1,43 @@
+.special-feed-card {
+    
+        
+    overflow: auto;
+    font-size: 0.9em;
+    margin-bottom: 2.5em;
+    /* background-color: rgb(151, 178, 212); */
```
Comment (pulsejet) [inline src/app/page/blog/blog.component.css]: Commented code is code smell

File: src/app/page/blog/blog.component.css [hunk 3]
```diff
@@ -1,13 +1,43 @@
...
+    border-radius: 10px 15px 15px 10px;
+    color: #fff;
+	padding: 8px;
+    font-weight: bold;
```
Comment (pulsejet) [inline src/app/page/blog/blog.component.css]: Wrong indentation

File: src/app/page/blog/blog.component.css [hunk 4]
```diff
@@ -1,13 +1,43 @@
...
+@keyframes blinkingBackground{
+    0%		{ background-color: #b6c3cf;}
+    /* 25%		{ background-color: #98b0da;} */
+    100%		{ background-color: #85b3ff;}
+    /* 75%		{ background-color: #6ba2ff;} */
+    /* 100%	{ background-color: #5e99ff;} */
+}
+    		
+
+
```
Comment (pulsejet) [inline src/app/page/blog/blog.component.css]: More newlines...

**Review by pulsejet — CHANGES_REQUESTED** [review]
Just some comments on code style, upto the maintainer to enforce or not.

Just curious, what are you trying to do here?

File: src/app/page/blog/blog.component.html [hunk 5]
```diff
@@ -18,8 +18,12 @@
 
 <div *ngIf="feed" class="main-div">
   <ng-container *ngFor="let item of feed">
-    <div class="feed-card">
-        <div class="title">{{ item.title }}</div>
+    <div [ngClass]="{
+                 'feed-card': item.pin_unpin== 0,
```
Comment (e-aakash) [inline src/app/page/blog/blog.component.html]: You may have to update this based on renamed value in backend. (`pin_unpin` -> `pinned`)

**Comment by e-aakash** [comment]
> Just curious, what are you trying to do here?

@pulsejet : Placement blog has introduced a new feature, for pinning certain blog. They asked for similar feature in instiapp blogs as well. The backend change was introduced in https://github.com/wncc/instiapp-api/pull/460 and https://github.com/wncc/instiapp-api/pull/461 
This pr is for visual highlighting the pinned blog post in webapp.

**Pull request closed without merging**